docs/source/api_commands.rst

```diff
@@ -194,10 +225,20 @@ carpet_clean_mode
 
 Get: get_carpet_clean_mode
```

Comment:
Would be nice if for all carpet clean modes we could say what each number means

Reply:
Agreed. I'm planning on adding more as I learn what each command does. Hoping other people will be able to help out as well, but just adding as I go along.